corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -653,7 +653,7 @@ hqDefine('geospatial/js/models', [
                 contentType: "application/json; charset=utf-8",
                 success: function (ret) {
                     if (!ret.success) {
-                        return alertUser.alert_user(message, 'danger');
+                        return alertUser.alert_user(gettext(ret.message), 'danger');
```

Comment:
will gettext really work for this since this will be dynamic/populated from response?

Reply:
Can we not just omit `gettext` entirely here? The text is already being translated in the back-end view.

Reply:
Thanks for catching this. 
@zandre-eng thats correct , we don't even need it. Resolved in https://github.com/dimagi/commcare-hq/pull/34717/commits/19ef6b9a32a60b81a8de0bdd4e27f7afc0cd228a